worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageUpdateConfig.java

```diff
@@ -18,4 +18,6 @@
 
     // Number of days that job is running
     private final int stuckHours;
+
+    private final boolean override;
```

Comment:
Use an ENUM instead.  That would give multiple options for operating modes - like skip refresh of current month on job submission.

Reply:
@nflearl can we instead put this off to another ticket?

Reply:
Sure, just comment with the ticket number.

Reply:
AB2D-3787